packages/cli/generators/model/index.js

```diff
@@ -200,7 +288,11 @@ module.exports = class ModelGenerator extends ArtifactGenerator {
       this.artifactInfo.outFile,
     );
 
-    const modelTemplatePath = this.templatePath('model.ts.ejs');
+    this.artifactInfo.isModelBaseBuiltin = BASE_MODELS.includes(
+      this.artifactInfo.modelBaseClass,
+    )
+      ? true
+      : false;
```

Comment:
Why are these true/false values needed? Doesn't `include` already return a boolean? If it does not, then you case add `!!` before the call to convert a value to a boolean.

```ts
!!BASE_MODELS.include(/*...*/)
```

Reply:
you were right, I simplified it. 👍